examples/tv-casting-app/tv-casting-common/src/ConversionUtils.cpp

```diff
@@ -18,27 +18,30 @@
 
 #include "ConversionUtils.h"
 
-CHIP_ERROR ConvertToDiscoveredNodeData(TargetVideoPlayerInfo * inPlayer, chip::Dnssd::DiscoveredNodeData & outNodeData)
+CHIP_ERROR ConvertToDiscoveredNodeData(TargetVideoPlayerInfo * inPlayer, chip::Dnssd::DiscoveredNodeData & outDiscNodeData)
 {
     if (inPlayer == nullptr)
         return CHIP_ERROR_INVALID_ARGUMENT;
+    
+    outDiscNodeData.Set<chip::Dnssd::CommissionNodeData>();
```

Comment:
We should make `Variant::Set` return the instantiated type like `std::variant::emplace` does to avoid this 'set followed by get' sequence (but not necessarily in this PR).

Reply:
Marking resolved as this is not necessary for this PR.